index.html

```diff
@@ -1,70 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" type="text/css" href="app.css" />
-    <title>Document</title>
-  </head>
-  <body>
-    <div class="container">
-      <div class="box">
-        <div class="boxtop">
-          <div class="boxtop-text">G</div>
-        </div>
-        <div class="box-text">
-          <div>Paying</div>
-          <div>Geolife agritech pvt ltd</div>
-          <div>rzp.pay/geolifeagritech</div>
-        </div>
-        <div class="input-amount">
-          <span class="currency">₹</span>
-          <!-- <img src="r.svg" /> -->
-          <input
-            type="text"
-            class="dynamic-input"
-            placeholder="0"
-            style="width: 25px"
-            maxlength="15"
-          />
-        </div>
-
-        <div class="comment">
-          <input type="text" placeholder="Comments (if any)" />
-        </div>
-        <button class="button">
-          <div class="button-text">Pay ₹0</div>
-        </button>
-        <div class="warning">
-          <img src="Union.svg" />
-          <div>This link is not activated for payments yet</div>
-        </div>
-      </div>
-      <div class="footer">
-        <div>
-          <img src="RZP logo.svg" id="desktop-image" />
```

Comment:
write alt attribute as well as it helps in case if the image cannot be displayed.